### PR TITLE
fix: properly size icons in distribute/align

### PR DESCRIPTION
**Pull request by barmac**

### CODE REVIEW

**Review by marstamm — APPROVED** [review]
Works on demo page 🚀

File: assets/bpmn-js.css [hunk 1]
```diff
@@ -129,15 +129,14 @@
...
-
   padding: 6px 8px;
 }
 
 [data-popup="align-elements"] .djs-popup-body .entry img {
-  height: 100%;
-  width: 100%;
+  display: block;
+
+  height: 20px;
```
Comment (nikku) [inline assets/bpmn-js.css:138]: Is there no other way than hardcoding (to the parent height)?

For me this looks like duplicate magic numbering. We already define the entry height in diagram-js (cf. https://github.com/bpmn-io/diagram-js/blob/develop/assets/diagram-js.css#L512).

Comment (barmac) [reply assets/bpmn-js.css:138]: Hmm, perhaps we could just use `1em` here. We need to set the image size explicitly because otherwise it shrinks.

Comment (barmac) [reply assets/bpmn-js.css:138]: But then the em value is also a magic number as it uses the line-height / font-size ratio 🙈

Comment (nikku) [reply assets/bpmn-js.css:138]: Isn't the issue that `width: 100%` does not mean a thing? I#ll have a look.

**Pull request merged**